src/Keboola/CsvMap/Mapper.php

```diff
@@ -71,38 +71,46 @@ private function expandShorthandDefinitions()
         }
     }
 
+    public function parse(array $data, array $userData = [])
+    {
+        // Create a file, even if there is no data
+        $this->getResultFile();
+        
+        foreach ($data as $row) {
+            $this->parseRow($row, $userData);
+        }
+    }
+
     /**
-     * @param array $data
+     * @param object|mixed $row
      */
-    public function parse(array $data, array $userData = [])
+    public function parseRow($row, array $userData = [])
```

Comment:
2. `Added public `parseRow` method", so single row can be parsed too.`

 Doteraz bola iba metoda `parse`, ktora parsuje pole (`rows`), no ja potrebujem parsovat po jednom riadku, preto som pridal metodu `parseRow`, ... aby som z toho nemusel robit pole s jednym prvkom.